server/actions/actionsRouter.js

```diff
@@ -0,0 +1,117 @@
+const express = require("express");
+const actionsDb = require("../data/helpers/actionModel");
+const projectsDb = require("../data/helpers/projectModel");
+
+const router = express.Router();
+
+// Get Single Action
+router.get("/:actionId", (req, res) => {
+  const { actionId } = req.params;
+  actionsDb
+    .get(actionId)
+    .then(action => {
+      if (action) {
+        res.status(200).json(action);
```

Comment:
You could replace this `magic number` as Luis calls them w/ a status code object. I've never used that method but am going to start on my next project.